builder/src/merge.rs

```diff
@@ -29,6 +29,20 @@ use super::{
 pub struct Merger {}
 
 impl Merger {
+    fn get_string_from_list(
```

Comment:
Ok, so maybe we can name the option `original-blob-ids` ? And the option help message should be `original blob id list separated by comma, it may usually be a sha256 hex string`.

Reply:
fixed